src/generators/date/index.ts

```diff
@@ -25,8 +31,9 @@ const createDateGenerator = (chance: Chance.Chance) => (
   }
 
   if (to == null) {
-    // ¯\_(ツ)_/¯
-    to = 'Tue Dec 31 2019 16:00:00 GMT-0800 (PST)';
+    // When to is null set some value that is never less than `from`
+    const toTime = integer(from.getTime(), maxTimeMilliseconds);
```

Comment:
We're using `getTime()` and `valueOf()` in the same method, but they do the same thing on `Date`. Can we use one or the other consistently?

Reply:
This is no longer necessary now that I addressed the comment below. I removed this unnecessary indirection here, and the `getTime` call was also removed.